src/nibetaseries/interfaces/nilearn.py

```diff
@@ -95,3 +94,12 @@ def _run_interface(self, runtime):
         self._results['correlation_fig'] = viz_mat_path
 
         return runtime
+
+
+def _fisher_r_to_z(x):
+    import numpy as np
+    # correct any rounding errors
+    # correlations cannot be greater than 1.
+    x[x > 1.] = 1.
+
+    return (np.log(1 + x) - np.log(1 - x)) * 0.5
```

Comment:
This can be replaced with just `np.arctanh(x)`.

Is the goal here to make the coefficients normally distributed, or do you want Z-statistics (for significance testing)?

Reply:
Thanks! You are correct, I can replace that.

The goal is to make the coefficients normally distributed, would you agree this is a reasonable method to do this or are there potentially better alternatives?

Reply:
Sorry, yes. I just didn't know if the z-values needed to be converted into test statistics as well (i.e., multiplied by `np.sqrt(n_trials - 3)`).

Reply:
no need to apologize, :laughing:  I was just curious. It may be worth considering how to include the confidence/reliability of the correlation we calculate (in another pull request since even including this code change is a bit off topic, but that is my fault).